src/detect-dns-query.c

```diff
@@ -92,14 +92,9 @@ void DetectDnsQueryRegister (void)
     DetectAppLayerInspectEngineRegister("dns_request",
             ALPROTO_DNS, SIG_FLAG_TOSERVER, 1,
             DetectEngineInspectDnsRequest);
-    DetectAppLayerInspectEngineRegister("dns_response",
```

Comment:
removing this will break other things I fear, thinking mostly about lua scripts

Reply:
It is just moved to detect-dns-response.c, I think that is more correct then leaving it in detect-dns-query.c, do you want me to move it back to detect-dns-query.c ?